ResourceManager/Xliff/TranslationUtil.cs

```diff
@@ -298,7 +299,7 @@ private static bool IsTranslatable(this Assembly assembly)
         public static Dictionary<string, List<Type>> GetTranslatableTypes()
         {
             Dictionary<string, List<Type>> dictionary = new();
-            foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
+            foreach (Assembly assembly in AppDomain.CurrentDomain.GetAssemblies().Where(p => !p.IsDynamic))
```

Comment:
This might cause why Forms of plugins are not translated anymore.

Reply:
No, this is the fix for https://ci.appveyor.com/project/gitextensions/gitextensions/builds/45774723#L346
```
System.NotSupportedException : The invoked member is not supported in a dynamic assembly.
   at System.Reflection.Emit.InternalAssemblyBuilder.get_Location()
   at ResourceManager.Xliff.TranslationUtil.GetTranslatableTypes() in C:\projects\gitextensions\ResourceManager\Xliff\TranslationUtil.cs:line 309
   at GitUITests.TranslationTest.CreateInstanceOfClass() in C:\projects\gitextensions\UnitTests\GitUI.Tests\TranslationTest.cs:line 37
```